External/FEXCore/Source/Interface/Core/Core.cpp

```diff
@@ -636,7 +683,62 @@ namespace FEXCore::Context {
     Thread->LookupCache->AddBlockMapping(Address, Ptr, Start, Length);
   }
 
+  void Context::SMCMarkReadOnly(uint64_t Start, uint64_t Length) {
+    auto Base = Start & FHU::FEX_PAGE_MASK;
```

Comment:
Could you add `const` to unmodified variables like this that span long functions?

Reply:
Done